tests/plugins/tool/pylint_tool_plugin/test_pylint_tool_plugin.py

```diff
@@ -70,6 +72,21 @@ def test_pylint_tool_plugin_scan_valid():
     assert len(issues) == 2
 
 
+def test_pylint_tool_plugin_scan_valid_max_cpu_cores():
+    """Integration test: Make sure the pylint output hasn't changed when using max CPU cores."""
+    max_procs = multiprocessing.cpu_count()
```

Comment:
I usually see things called `max_procs` referring to a process count, not a cpu count.  Though since presumably pylint is spawning one process per cpu, I guess it's still valid.   I'm fine as-is, just pointing that out.

Reply:
Since the top-level argument for Statick is `--max-procs` I'm going to leave this as-is for now. If starting from scratch I would agree with you and change to `max_cpus` or similar.